README.md

```diff
@@ -303,7 +303,7 @@ export const authProvider = new MsalAuthProvider(
 import { authProvider } from './authProvider';
 
 const request = async url => {
-  const token = await authProvider.getAccessToken();
```

Comment:
This actually should be `getAccessToken()`, it's the following lines that should be modified. Looks like a copy-paste issue with the example. The bearer token should pass `getAccessToken().accessToken` I believe. Most APIs authorize via access tokens, not IdTokens.